Makefile

```diff
@@ -59,11 +59,19 @@ HELM_CHART_LINT_ARGS_crossplane = --set nameOverride='',imagePullSecrets=''
 # Due to the way that the shared build logic works, images should
 # all be in folders at the same level (no additional levels of nesting).
 
-REGISTRY_ORGS = docker.io/crossplane
+REGISTRY_ORGS = docker.io/crossplane xpkg.upbound.io/crossplane
 IMAGES = crossplane
 OSBASEIMAGE = gcr.io/distroless/static:nonroot
 -include build/makelib/imagelight.mk
 
+# ====================================================================================
+# Setup Docs
+
+SOURCE_DOCS_DIR = docs
+DEST_DOCS_DIR = content
+DOCS_GIT_REPO = https://$(DOCS_GIT_USR):$(DOCS_GIT_PSW)@github.com/crossplane/docs.git
+-include build/makelib/docs.mk
+
```

Comment:
@nullable-eth looks like some of the recently removed docs functionality got re-added here. A rebase should fix it 👍🏻

Reply:
Sorry my fork was not up to date.